mypyc/lib-rt/getargs.c

```diff
@@ -1197,6 +1200,12 @@ vgetargskeywords(PyObject *args, PyObject *kwargs, const char *format,
         }
     }
 
+    if (*format == '%') {
```

Comment:
Document somewhere in this file what `%` means, since it's non-standard. Probably also worth discussing the expected arguments and what NULL means.